circuit_knitting/forging/__init__.py

```diff
@@ -43,6 +44,7 @@
 from .entanglement_forging_operator import EntanglementForgingOperator
 from .entanglement_forging_ground_state_solver import (
     EntanglementForgingGroundStateSolver,
+    EntanglementForgingResult,
```

Comment:
Shouldnt have to go all the way into the EFGSSolver to get the main result type of this tool. Adding it here